presto-raptor/src/test/java/com/facebook/presto/raptor/metadata/TestShardPredicate.java

```diff
@@ -108,7 +108,7 @@ public void testBucketNumber()
     {
         TupleDomain<RaptorColumnHandle> tupleDomain = withColumnDomains(ImmutableMap.of(
                 bucketNumberColumnHandle("test"),
-                create(SortedRangeSet.copyOf(INTEGER, ImmutableList.of(equal(INTEGER, 1L), equal(INTEGER, 2L))), false)));
```

Comment:
It shouldn't have been there in the first place because shard predicate does not handle multiple ranges. This case will work when we implement that.